cfgov/unprocessed/js/modules/util/validators.js

```diff
@@ -0,0 +1,114 @@
+/* ==========================================================================
+   Validators
+
+   Various utility functions to check if a field contains a valid value
+   ========================================================================== */
+
+'use strict';
+
+// Required Modules
+var ERROR_MESSAGES = require( '../../config/error-messages-config' );
+var typeCheckers = require( '../../modules/util/type-checkers' );
+
+/**
+ * date Determines if a field contains a valid date
+ *
+ * @param {object} field         Field to test
+ * @param {object} currentStatus A previous tested status for the field
+ * @returns {object} empty if the field passes, msg and type if it failed
+ */
+function date( field, currentStatus ) {
+  var status = currentStatus || {};
+  var dateRegex =
+    /^\d{2}$|^\d{4}$|^\d{2}\/(?:\d{4}|\d{2})$|^\d{2}\/\d{2}\/\d{4}$/;
+  if ( field.value && dateRegex.test( field.value ) === false ) {
+    status.msg = status.msg || '';
+    status.msg += ERROR_MESSAGES.DATE.INVALID;
+    status.date = false;
+  }
+  return status;
+}
+
+/**
+ * email Determines if a field contains a email date
+ *
+ * @param {object} field         Field to test
+ * @param {object} currentStatus A previous tested status for the field
+ * @returns {object} empty if the field passes, msg and type if it failed
+ */
+function email( field, currentStatus ) {
+  var status = currentStatus || {};
+  var regex = [
+    '^[a-z0-9\u007F-\uffff!#$%&\'*+\/=?^_`{|}~-]+(?:\.[a-z0-9',
+    '\u007F-\uffff!#$%&\'*+\/=?^_`{|}~-]+)*@(?:[a-z0-9]',
+    '(?:[a-z0-9-]*[a-z0-9])?\.)+[a-z]{2,}$'
```

Comment:
As mentioned in the prior lint PR, could this just be a straight-up string? It's an array that's turned into a string with `join()` so the array bits be removed and the commas replaced with `+`?